package.json

```diff
@@ -16,13 +16,14 @@
     "babel-preset-stage-2": "^6.22.0",
     "catalog": "^3.1.2",
     "core-js": "^2.4.1",
+    "create-react-class": "^15.6.2",
     "cz-conventional-changelog": "^2.0.0",
     "glamor": "^2.20.37",
     "husky": "^0.14.3",
     "prop-types": "^15.5.7",
-    "react": "^15.4.2",
+    "react": "^16.0.0",
     "react-autocomplete": "^1.4.1",
-    "react-dom": "^15.4.2",
+    "react-dom": "^16.0.0",
     "react-maskedinput": "^3.3.4",
```

Comment:
you can upgrade to [4.0.0](https://github.com/insin/react-maskedinput/releases/tag/v4.0.0). Not a React 16 release but no longer uses `React.PropTypes` and `React.createClass` – should be compatible.